x-pack/plugins/actions/server/actions_client/actions_client.test.ts

```diff
@@ -2743,270 +2709,6 @@ describe('update()', () => {
 });
 
 describe('execute()', () => {
-  describe('authorization', () => {
```

Comment:
we should keep the RBAC authorization tests right?

Reply:
Yes you're right! I prob shouldn't have removed all of them

Reply:
Resolved in this commit [83caa27](https://github.com/elastic/kibana/pull/197719/commits/83caa278560ace58788b504fc7de7503cca9808c)